tests/framework/installer/ceph_installer.go

```diff
@@ -761,6 +761,7 @@ func (h *CephInstaller) GatherAllRookLogs(testName string, namespaces ...string)
 	for _, namespace := range namespaces {
 		h.k8shelper.GetLogsFromNamespace(namespace, testName, testEnvName())
 		h.k8shelper.GetPodDescribeFromNamespace(namespace, testName, testEnvName())
+		h.k8shelper.GetEventsFromNamespace(namespace, testName, testEnvName())
```

Comment:
If we gather all the events in the namespace, I think it will be very verbose. Do you have an example of the output to see how useful it is? The pod describe will already gather the events related to the pods, so that has been the most useful info until now.

Reply:
Thank you for your comment!

> Do you have an example of the output to see how useful it is?

I think this is useful when pods are not created for some reasons. 
For example, if you specify non-existing service account for the pod template in some deployment resource, the reason why the pods are not created is recorded as the related replicaset's event.

Reply:
Printing all the events is just very verbose. In the past we actually did it, but then turned it off. Can we filter to certain events? Or can you show the example output? Maybe it's more readable than what we had in the past.

Reply:
I think it would be fine to leave the file unfiltered. We will only want to reference it when other methods are failing, and I think it's okay to parse by doing text search or scanning manually in those extreme cases.

Reply:
Agreed, in a separate file will be ok. The reason we disabled the events in the past was that they were included inline in the test log, which was too overwhelming.